unused/ui/TerminalController.java

```diff
@@ -0,0 +1,34 @@
+package ui;
+
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+
+//@@author NicholasTanYY-unused
+// Reason: Initially wanted to have the screen cleared everytime the user presses enter, 
+// to make the UI cleaner. However using keyboard inputs worked differently across different 
+// terminals and different OS. This became too troublesome to manage.
+public class TerminalController {
+
+    private static Robot robot;
```

Comment:
Thanks for adding an unused folder to log and store all previous contributions from our side